script.js

```diff
@@ -1,4 +1,140 @@
-var main = function (input) {
-  var myOutputValue = 'hello world';
+/* Create a basic version of Scissors Paper Stone where the user inputs one of "scissors", "paper", or "stone", the program internally randomly chooses scissors, paper, or stone, and the program outputs whether the user won, the program won, or it's a draw.
+Rules: scissors beats paper, paper beats stone, and stone beats scissors. If both parties choose the same object, it's a draw. */
+
+// Add state to your program such that it keeps track of the number of times the user has won and the number of times the computer has won. Output this win-loss record in a format you like in the program output. You can also output the number of draws and/or each party's winning percentage if you'd like.
+
+// Ensuring the input is correct
+
+// Pseudo: Creating variable for scissors, paper and stone and also the results.
+var scissors = "scissors";
+var paper = "paper";
+var stone = "stone";
+var loseMessage = "You lose! Bummer.";
+var winMessage = "Congratulations, you win!";
+var drawMessage = "It's a tie.";
+var invalid =
+  "Invalid input. You can only choose scissors, paper or stone. Please try again. ";
+var numbOfGames = NaN || 0;
```

Comment:
It would be good to add comments for syntax that isn't commonly used so that people can understand it easily